### ISSUE TITLE
Eye tracking (for OSC etc)

### ISSUE THREAD

**Issue by shinyquagsire23** [alvr-org/alvr-visionos#21]
Possibly relevant docs:
https://developer.apple.com/documentation/compositorservices/4082136-cp_drawable_get_rasterization_ra
https://developer.apple.com/documentation/metal/mtlrasterizationratemap

The rasterization map might leak some basic eye tracking? Needs investigation. And maybe pestering Apple for an actual permission + API if the above doesn't work.

**Comment by xuhao1** [comment]
@shinyquagsire23  In accessibility, we can enable  pointer control with eyes, and a white pointer will show on screen. Can we obtain the position of this pointer? It’s like a mouse but it’s on the position where are are looking at.

**Comment by shinyquagsire23** [comment]
I wish lol, none of the accessibility stuff even seems to work in immersive mode, keyboard-based pinch events report 0,0,0 as their gaze ray, and we only have one event handler accessible bc there's no windows

**Comment by xuhao1** [comment]
@shinyquagsire23 I can still see a mouse pointer in immersive mode. Will test if it have correct mouse input.

**Comment by shinyquagsire23** [comment]
yes it's visible but there's no flat plane to get the movements from. Frankly I doubt even flat plane apps get them.

**Comment by xuhao1** [comment]
Can we put a huge cube to capture it?

**Comment by shinyquagsire23** [comment]
Technically maybe, but again the flip side is like, I don't know if those movements can even be sent to panels at all, *and* Apple doesn't allow programmatically moving windows, so the user would have to position them by hand :/
